tests/README.md

```diff
@@ -142,3 +142,23 @@ To run specific tests inside a suite:
 ```console
 _output/tests/linux_amd64/integration -test.v -test.timeout 1800s -test.run CephSmokeSuite -testify.m TestARookClusterInstallation_SmokeTest
 ```
+
+### To run tests on OpenShift environment
+
+- You have to setup OpenShift environment by your own and export KUBECONFIG before executing the tests.
+- Make sure `oc` executable file is in your PATH.
+- Currenttly we are enabling only `CephSmokeSuite`  on OpenShift environment.
+- To start the test we need to set few environment variables :-
+```
+export TEST_PLATFORM=openshift
+export TEST_ENV_NAME=openshift
```

Comment:
Why do we need TEST_ENV_NAME? Can we just use TEST_PLATFORM instead of a new var?

Reply:
Actually, removing `TEST_PLATFORM` instead, we are using  `TEST_ENV_NAME` in past also we this var in Jenkins files.